lib/git.js

```diff
@@ -47,12 +48,13 @@ exports.getCommits = function () {
     }
 
     commit.type = parsed[1].toLowerCase();
-    commit.category = parsed[3];
+    commit.category = parsed[3] || '';
```

Comment:
Actually, scratch that. It's a step in the right direction, but we'll also need some changes mentioned in #6. You don't have to worry about those though.